### PR TITLE
Added open design section

### PR DESCRIPTION
**Pull request by simon332114**
New open design section and relevant documentation

### CODE REVIEW

**Comment by simon332114** [comment]
Mistake in commit 97e7764 it was the code config that I modified, not design

**Comment by ShadeWyrm** [comment]
@simon332114 - please resolve conflicts with the form.yml file.

Also note, that the Project Licenses Dropdown doesn't work.

**Comment by simon332114** [comment]
The licenses dropdown works on my end, so I'm not sure what's the problem with that one

**Comment by ShadeWyrm** [comment]
> The licenses dropdown works on my end, so I'm not sure what's the problem with that one

It was a user error on my part! Submodule didn't download for some reason. Restarting Visual Code fixed it.

File: _data/software/tomcat.yml [hunk 1]
```diff
@@ -1,38 +0,0 @@
----
```
Comment (ShadeWyrm) [inline _data/software/tomcat.yml]: Please restore this file.

**Comment by ShadeWyrm** [comment]
Another issue : https://github.com/canada-ca/ore-ero/pull/1011/files#submit-review

The "designTypes" fields are blank.

**Comment by simon332114** [comment]
Should be fixed now

**Pull request merged**